bin/jpm

```diff
@@ -69,6 +69,12 @@ program
 program
   .command("xpi")
   .description("Bundle the addon into an .xpi file")
+  .option("--xpiPath <output_dir>", "Where to write the generated xpi",
```

Comment:
oh, I see what you were doing. Hmm, `options.xpiPath` was a poor choice of names to indicate a destination directory :) Check the [commander docs](https://github.com/tj/commander.js) to see what the best way is to turn the `--dest-dir` value into `options.xpiPath`.